api/src/test/java/com/epam/pipeline/manager/issue/IssueApiServiceTest.java

```diff
@@ -16,6 +16,7 @@
 
```

Comment:
Could you please delete all the test cases in the class that is covered by the new test class?

Additionally check if the remaining test cases (probably only `testIssueOwnerCRUD`) are covered by `IssueManagerTest`. If not please try to migrate the remaining methods to `IssueManagerTest`.